stable/coredns/templates/serviceaccount-autoscaler.yaml

```diff
@@ -18,4 +18,9 @@ metadata:
 {{- if .Values.customLabels }}
 {{ toYaml .Values.customLabels | indent 4 }}
 {{- end }}
+{{- if .Values.autoscaler.image.pullSecrets }}
+imagePullSecrets:
+{{- range .Values.autoscaler.image.pullSecrets }}
+  - name: {{ . }}
+{{- end }}
```

Comment:
Is this missing a second {{- end }}?

Reply:
Fixed and rebased the commits (https://github.com/coredns/helm/pull/36/commits/1df87bc1e980491ac78e6d49a4f287039aa4b333)